.github/workflows/main.yml

```
@@ -136,13 +136,13 @@ jobs:
 
       - name: Save docker image
         run: |
-          docker save job-server | gzip > /tmp/job-server.tar.gz
+          docker save job-server | zstd --fast -o /tmp/job-server.tar.zst
 
       - name: Upload docker image
         uses: actions/upload-artifact@v4
         with:
             name: job-server-image
-            path: /tmp/job-server.tar.gz
+            path: /tmp/job-server.tar.zst
```

thought: You might get a further improvement from:

```suggestion
            path: /tmp/job-server.tar.zst
            # Disable compression; the file is already compressed
            compression-level: 0
```

---

No harm in adding it, so I will!